AdminClient/src/app/customers/customers.component.ts

```diff
@@ -0,0 +1,55 @@
+import {Component, Inject, OnInit, Output} from '@angular/core';
```

Comment:
@HarryPi should this be in its own module?

Reply:
A component should be its own module when its a top level component i.e not a child of any component besides the app component. Giving the definition as i dont know the structure here.